### PR TITLE
Update uri-templates to 0.1.5

### PR DESCRIPTION
**Pull request by jtamayo**

### CODE REVIEW

**Comment by vvakame** [comment]
@Bartvds could you review this PR?

File: uri-templates/uri-templates.d.ts [hunk 1]
```diff
@@ -1,18 +1,19 @@
-// Type definitions for uri-templates 0.1.2
+// Type definitions for uri-templates 0.1.5
 // Project: https://github.com/geraintluff/uri-templates
-// Definitions by: Bart van der Schoor <https://github.com/Bartvds>
+// Definitions by: Juan M Tamayo <https://github.com/jtamayo>
```
Comment (vvakame) [inline uri-templates/uri-templates.d.ts]: please keep original author. add your name below.

File: uri-templates/uri-templates-0.1.2.d.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+// Type definitions for uri-templates 0.1.2
```
Comment (thasner) [inline uri-templates/uri-templates-0.1.2.d.ts]: is this just here for back-compat? did you also make modifications?

File: uri-templates/uri-templates.d.ts [hunk 3]
```diff
@@ -1,18 +1,19 @@
-// Type definitions for uri-templates 0.1.2
+// Type definitions for uri-templates 0.1.5
 // Project: https://github.com/geraintluff/uri-templates
-// Definitions by: Bart van der Schoor <https://github.com/Bartvds>
+// Definitions by: Bart van der Schoor <https://github.com/Bartvds>, Juan Tamayo <https://github.com/jtamayo>
 // Definitions: https://github.com/borisyankov/DefinitelyTyped
 
-declare module 'uri-templates' {
-	function utpl(template: string): utpl.URITemplate;
+declare class UriTemplate {
```
Comment (thasner) [inline uri-templates/uri-templates.d.ts]: consider using an `interface` to define the ability to call a constructor with the `new` keyword.

also, consider using a common name-spacing pattern where all definitions are scoped within a `UriTemplates` namespace
```
declare module UriTemplates {
    interface UriTemplate {
        new(template: string): UriTemplate;
        (template: string): UriTemplate;
        // etc
    }
}

declare module "uri-templates" {
  export = UriTemplates.UriTemplate;
}
```

**Comment by RyanCavanaugh** [comment]
Hello and thank you for your contribution!

Due to an excessively long queue of pull requests that have become stale, we are "declaring bankruptcy" and closing all PRs opened before May 1, 2016. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

Going forward, we are committing to review or merge all PRs on a regular basis so this bankruptcy will not occur again. We apologize for the incovenience and hope you will continue to contribute to DefinitelyTyped in the future.

Thanks
The TypeScript and DefinitelyTyped teams

**Pull request closed without merging**